src/controllers/authController.js

```diff
@@ -19,17 +22,89 @@ export default class AuthController {
   }
 
   /**
- *
+   * @param {object} req request body
+   * @param {object} res response body
+   * @returns  {object} message
+   */
+  static async forgotPassword({ body: { email } }, res) {
+    try {
+      const data = await AuthService.forgotPassword(email);
+
+      res.status(200).json({ status: 'success', data });
+    } catch ({ message: error }) {
+      res.status(400).json({ status: 'error', error });
+    }
+  }
+
+  /**
+   * @param {object} req request body
+   * @param {object} res response body
+   * @returns  {object} message
+   */
+  static async resetPassword({ params: { token }, body: { password } }, res) {
+    try {
+      const message = await AuthService.resetPassword(token, password);
+
+      res.status(200).json({ status: 'success', message });
+    } catch ({ message: error }) {
+      res.status(400).json({ status: 'error', error });
+    }
+  }
+
+  /**
  * @param {object} res - response object
  * @return {object} - user data and status code
  */
   static async signupUser({ body }, res) {
     try {
       const data = await AuthService.signup(body);
+      const url = `${process.env.APP_URL}/verify?token=${data.token}`;
```

Comment:
Call the authService verification function here to replace line `61` to line `70`